server/@types/dto/index.d.ts

```diff
@@ -18,7 +18,7 @@ declare module 'dto' {
     title: string
     status: string
     steps: Array<UpdateStepDto>
-    targetCompletionDate?: string
+    targetCompletionDate?: Date
```

Comment:
But this needs to be back as a string if we can (unless you found a compilation or test error that made you change this?)